.env.example

```diff
@@ -19,6 +19,9 @@ MAGENTO_ADMIN_PASSWORD=
 #SELENIUM_PROTOCOL=http
 #SELENIUM_PATH=/wd/hub
 
+#*** Browser for running tests, default chrome. Uncomment and change if you want to run tests on another browser (ex. firefox).
+#BROWSER=chrome
```

Comment:
Please uncomment `BROWSER`
Let it be always set by default to chrome

Reply:
Fixed